spec/controllers/conferences_controller_spec.rb

```diff
@@ -26,6 +26,13 @@
     end
   end
 
+  describe 'GET #current' do
+    it 'redirects to first conference splashpage' do
+      current = Conference.first
+      expect(response).to redirect_to(conference_path(current.short_title))
```

Comment:
this test will only test that the action redirects to the splashpage for the first conferences. That is the only thing this `conference#current`  is supposed to do.

This is my first ever time trying to write tests, so I could use a bit of help please.